static/src/stylesheets/module/identity/_identity.scss

```diff
@@ -422,18 +414,32 @@
     @include fs-bodyCopy(1);
 }
 
+.saved-content__meta {
+    padding-top: $gs-baseline;
+    text-align: right;
+
+    .button {
```

Comment:
I suppose this is to get around some specificity issues. Could we rather use '.saved-content__button'? So the changes will be visually coupled only within the `saved-content` object.